recipes/boltons/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: boltons
-  version: {{ environ['GIT_DESCRIBE_TAG'] }}
+  version: {{ GIT_DESCRIBE_TAG }}
```

Comment:
This might not throw an error, but it should result in an empty version instead. This is because with --dirty, the environment variable can't be known, right? I would suggest to either just copy paste the git_rev value or define it via jinja.